### PR TITLE
Bug fix in csvreads. (Bug #1423)

### PR DESCRIPTION
**Pull request by AdithyaPediredla**
[Describe your changes and what you've already done to test it.]

#### Check all that apply (change to `[x]`)
- [x] All changes meet [libigl style-guidelines](https://libigl.github.io/style-guidelines/).
- [ ] Adds new .cpp file.
- [ ] Adds corresponding unit test.
- [ ] Adds corresponding python binding.
- [x] This is a minor change.

### CODE REVIEW

File: include/igl/readCSV.cpp [hunk 1]
```diff
@@ -30,8 +30,10 @@ IGL_INLINE bool igl::readCSV(
     std::istringstream iss(line);
     vector<Scalar> temp;
     Scalar a;
-    while (iss >> a)
+    char ch;
+    while (iss >> a >> ch)
       temp.push_back(a);
+    temp.push_back(a);
```
Comment (jdumas) [inline include/igl/readCSV.cpp]: This will not work if the csv is empty.

Comment (AdithyaPediredla) [reply include/igl/readCSV.cpp]: Thank you. I have changed line 36 to
`if(iss >> a)
    temp.push_back(a);
`
and merged into the same commit. 

Do I have to make another pull request?

Comment (jdumas) [reply include/igl/readCSV.cpp]: No just update your branch and you'll be fine. But I think your fix still doesn't cover all cases. I would change it to the following:
```cpp
while (iss >> a) {
  temp.push_back(a);
  if (!(iss >> ch)) {
    break;
  }
}
```

Comment (AdithyaPediredla) [reply include/igl/readCSV.cpp]: Ya, you are right again. I will fix it.

Comment (AdithyaPediredla) [reply include/igl/readCSV.cpp]: I fixed it. You can pull now.

**Pull request merged**